bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -1008,6 +1033,10 @@ impl EventTimelineItem {
     pub fn can_be_replied_to(&self) -> bool {
         self.0.can_be_replied_to()
     }
+
```

Comment:
Please add docs, I know it's annoying to duplicate them but the docs now are visible in Kotlin/Swift land so they do have a use.